lang/rust/avro/src/encode.rs

```
@@ -114,6 +115,12 @@ pub(crate) fn encode_internal<S: Borrow<Schema>>(
             &uuid.to_string(),
             buffer,
         ),
+        Value::BigDecimal(bg) => {
+            let result: Result<Vec<u8>, Error> = serialize_big_decimal(bg);
+            if let Ok(mut buf) = result {
+                buffer.append(&mut buf);
+            }
```

I think we don't need the Result. Or we must not ignore the error here.